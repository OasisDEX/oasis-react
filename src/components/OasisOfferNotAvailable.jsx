import React, { PureComponent } from 'react';
import { PropTypes } from 'prop-types';

import styles from './OasisOfferNotAvailable.scss';
import InfoBoxBody from './InfoBoxBody';
import InfoBox from './InfoBox';
import OasisIcon from './OasisIcon';


const propTypes = PropTypes && {};
const defaultProps = {};


class OasisOfferNotAvailable extends PureComponent {
  render() {
    return (
      <InfoBox color="danger" className={styles.base}>
        <InfoBoxBody>
          <div>
            <OasisIcon icon="failed"/>
          </div>
          <div>This order is <b>not available anymore</b></div>
        </InfoBoxBody>
      </InfoBox>
    );
  }
}

OasisOfferNotAvailable.displayName = 'OasisOfferNotAvailable';
OasisOfferNotAvailable.propTypes = propTypes;
OasisOfferNotAvailable.defaultProps = defaultProps;
export default OasisOfferNotAvailable;
