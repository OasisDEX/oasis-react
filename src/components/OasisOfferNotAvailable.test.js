/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';
import { shallow } from 'enzyme';
import OasisOfferNotAvailable from './OasisOfferNotAvailable';


describe('(Component) OasisOfferNotAvailable', () => {
  it('should render', () => {
    const props = {
      children: (
        <span>test</span>
      )
    };
    const wrapper = shallow(
      <OasisOfferNotAvailable {...props}/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
