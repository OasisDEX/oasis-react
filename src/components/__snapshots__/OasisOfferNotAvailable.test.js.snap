// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) OasisOfferNotAvailable should render 1`] = `
<InfoBox
  className=""
  color="danger"
  justifyContent="normal"
  size="md"
  vertical={false}
>
  <InfoBoxBody
    className=""
    color="default"
    vertical={false}
  >
    <div>
      Offer is not available anymore
    </div>
    <div>
      Will close shortly
    </div>
  </InfoBoxBody>
</InfoBox>
`;
